astropy/utils/iers/iers.py

```diff
@@ -462,7 +462,8 @@ def _combine_a_b_columns(cls, iers_a):
         #              Bull. A polar motion values
         # UT1Flag_A    IERS (I) or Prediction (P) flag for
         #              Bull. A UT1-UTC values
-        is_predictive = (table['UT1Flag_A'] == 'P') | (table['PolPMFlag_A'] == 'P')
+        is_predictive = ((table['UT1Flag_A'].filled() == 'P') |
+                         (table['PolPMFlag_A'].filled() == 'P'))
```

Comment:
Just in case the fill value ends up being ``P``, I think it would be better to do

```python
        is_predictive = ((table['UT1Flag_A'] == 'P').filled(False) |
                         (table['PolPMFlag_A'] == 'P').filled(False))
```